workspaces/analytics/tests/__snapshots__/EventBus.test.ts.snap

```diff
@@ -3,18 +3,21 @@
 exports[`Event Bus can listen for analytics events 1`] = `
 Object {
   "context": Object {
-    "clientAgent": "testing",
+    "apiName": "testingApiName",
+    "arch": "mips",
+    "clientAgent": "testingAgent",
     "clientId": "testing",
+    "clientSessionInstanceId": "testing",
+    "clientTimestamp": "testingTime",
     "platform": "furby",
-    "arch": "mips",
     "release": "1998",
-    "apiName": "testing",
-    "clientSessionInstanceId": "testing",
-    "clientTimestamp": "testing",
+    "source": "user",
   },
-  "data": Object {
-    "apiName": "Hello World API",
+  "event": Object {
```

Comment:
This is the main change in the snapshot, event and context are separated (since TrackingEventBase no longer contains context)